common/common-client/src/main/java/org/ow2/proactive/authentication/FileLoginModule.java

```diff
@@ -207,22 +219,37 @@ protected boolean logUser(String username, String password, boolean printErrorMe
      * @throws LoginException if login file is not found or unreadable.
      */
     private boolean authenticateUserFromFile(String username, String password) throws LoginException {
+
+
         Properties props = new Properties();
+        PrivateKey privateKey = null;
         try {
+            privateKey = getPrivateKey();
             FileInputStream stream = new FileInputStream(new File(loginFile));
             props.load(stream);
             stream.close();
+        } catch (KeyException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1166)